force_wfmanager/left_side_pane/results_pane.py

```diff
@@ -0,0 +1,50 @@
+from pyface.tasks.api import TraitsDockPane
+
+from traits.api import Instance
+
+from traitsui.api import View, UItem, VGroup
+
+from force_bdss.api import IFactoryRegistryPlugin
+from force_wfmanager.central_pane.result_table import ResultTable
+from force_wfmanager.central_pane.analysis_model import AnalysisModel
+
+
+class ResultsPane(TraitsDockPane):
+    """ Side pane which contains the WorkflowSettings (Tree editor for the
+    Workflow) and the Run button """
```

Comment:
Missing empty line after the docstring?

Reply:
Well it looks like me and Stefano did this for a lot of the docstrings! I'll fix them as I see them